test/google.spec.ts

```diff
@@ -2,13 +2,12 @@ import { browser } from 'protractor';
 
```

Comment:
delete blank lines at the start of a file

Reply:
![image](https://user-images.githubusercontent.com/38733280/102394255-7af69200-3fa7-11eb-93a9-547370a95e6c.png)

I think it is not necessary to delete line two, since it separates the imports from the code, why do you say that?

Reply:
my mistake! you're right